src/pages/Home.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useSearchParams, useNavigate } from 'react-router-dom';
+import Coin from '../components/Coin';
+import Header from '../components/Header';
+import Chart from '../components/Chart';
+import { FetchCoinsFunc } from '../redux/List/List';
+import '../styles/Home.css';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const coins = useSelector((state) => state.ListsReducer);
+  useEffect(() => {
+    dispatch(FetchCoinsFunc());
+  }, [dispatch]);
+
+  const [search, setSearch] = useSearchParams();
+
+  const handleDetails = (id) => navigate(`/details/${id}`);
+  return (
+    <div data-testid="intro-page" className="data-1">
+      <Header path="/" />
+      <Chart title1="Crypto" title2="Currencies" count={`${coins.length} Coins`} />
+
+      <div>
+
+        <input
+          className="coins-abt"
+          type="text"
+          value={search.get('filter') || ''}
+          placeholder="Search by Crypto name  eg : BNB"
+          onChange={(e) => {
+            const filter = e.target.value;
+            if (filter) {
+              setSearch({ filter });
+            } else {
+              setSearch({});
+            }
+          }}
+        />
+        <div className="coin-card-1">
+          {coins
+            .filter((coin) => {
+              const filter = search.get('filter');
+              if (!filter) return true;
+              const name = coin.name.toLowerCase();
+              return name.startsWith(filter.toLowerCase());
+            })
+            .map((coin) => (
+              <Coin
+                key={coin.id}
+                name={coin.name}
+                symbol={coin.symbol}
+                id={coin.id}
+                price={coin.priceUsd}
+                handleDetails={handleDetails}
+              />
+            ))}
+        </div>
```

Comment:
Nice and beautiful design so far, however, as stated in the project requirements, you are suppose to follow the design guidelines including: 

- [x] Colors (select one main color for your website).
- [x] Typography: font face, size and weight.
- [x] Layout: composition and space between elements.

There are some differences between yours and that of the actual design which you will need to fix.

1. The background of each card alternates between light and darker color along the column forming a checkered layout.
2. The position of the small arrow in the circle is at the top right corner and not the top middle. 

### Your design 


![Screenshot from 2023-02-18 12-46-12](https://user-images.githubusercontent.com/65233860/219867867-fc4cc934-8804-432a-96fa-52f8666cf104.png)


### The actual design 


![Screenshot from 2023-02-18 12-57-13](https://user-images.githubusercontent.com/65233860/219868038-e10adb4d-67a0-4e6b-a929-f67c8af1d6d5.png)